### PR TITLE
Cockpit 286

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-286.html

 - [x] update bodhi URLs for cockpit: https://bodhi.fedoraproject.org/updates/?packages=cockpit
 - [x] update bodhi URLs for c-machines: https://bodhi.fedoraproject.org/updates/?packages=cockpit-machines
 - [x] update bodhi URLs for c-podman: https://bodhi.fedoraproject.org/updates/?packages=cockpit-podman

### CODE REVIEW

**Comment by martinpitt** [comment]
cockpit is built everywhere, bodhi URLs can be updated. I added a TODO list to the description to keep track of releases and bodhi.

File: _posts/2023-02-22-cockpit-286.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 286, Cockpit-machines 284, Cockpit-podman 63
+author: garrett
+date: '2023-02-22'
+tags: cockpit
+slug: cockpit-286
+category: release
+summary: ''
```
Comment (garrett) [inline _posts/2023-02-22-cockpit-286.md:8]:
```suggestion
summary: 'Toggle resource visibility in the metrics page'
```

**Review by garrett — COMMENTED** [review]
Added summary and a suggestion for the description for the metrics toggle.

File: _posts/2023-02-22-cockpit-286.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 286, Cockpit-machines 284, Cockpit-podman 63
```
Comment (garrett) [inline _posts/2023-02-22-cockpit-286.md:2]: Title looks a bit long in the browser...

image

image

Perhaps we should drop Cockpit- from Machines and Podman?

```suggestion
title: Cockpit 286, Machines 284, and Podman 63
```

Comment (garrett) [reply _posts/2023-02-22-cockpit-286.md:2]: But then it does look like we're announcing Podman 63. Hm.

Comment (garrett) [reply _posts/2023-02-22-cockpit-286.md:2]: We usually announce the Cockpit release in the title and the others at the top and in the links at the bottom.

Comment (mvollmer) [reply _posts/2023-02-22-cockpit-286.md]: Changed.

File: _posts/2023-02-22-cockpit-286.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 286, Cockpit-machines 284, and
+Cockpit-podman 63:
+
+
+## Metrics page: toggle visibility of the resource usage graphs
```
Comment (garrett) [inline _posts/2023-02-22-cockpit-286.md:18]:
```suggestion
## Metrics: Toggle visibility of resource usage graphs
```

**Pull request merged**